### DISCUSSION TITLE
Setup Multiple Node for Elastic search on Wikijs

### DISCUSSION THREAD

**Discussion by Tai-DH** [requarks/wiki#7495] [Help / Questions] [answered]
Hi everyone, I am using Wiki.js to build my own help page based on the sample design below. I have two machines:

IP1: 10.0.0.1
IP2: 10.0.0.2
On each machine, I am hosting a Wiki.js server on port 3000 and Elasticsearch on port 9200. I have used some tricks to configure redundancy for the Wiki.js server by using Caddy Server and Network Load Balancing on Windows Server. However, I have no idea how to configure redundancy for the Elasticsearch host in the search engine settings. Are there any extended settings for Wiki.js that can support this?

**Comment by NGPixel** [comment, marked as answer]
Simply list all hosts, separated by a comma. e.g. `10.0.0.1:9200, 10.0.0.2:9200`
